trulens_eval/trulens_eval/pages/Evaluations.py

```diff
@@ -143,6 +145,29 @@
             with st.expander("Response", expanded=True):
                 st.write(response)
 
+            row = selected_rows.head().iloc[0]
+
+            st.header("Feedback")
+            for fcol in feedback_cols:
+                feedback_name = fcol
+                feedback_result = row[fcol]
+                feedback_calls = row[f"{fcol}_calls"]
+
+
+                def display_feedback_call(call):
+                    def highlight(s):
+                        return ['background-color: #4CAF50']*len(s) if s.result >= 0.5 else ['background-color: #FCE6E6']*len(s)
```

Comment:
Can you merge with Josh's https://github.com/truera/trulens/pull/145/files
and use the default_pass_fail_color_threshold constant?